src/elexmodel/models/BootstrapElectionModel.py

```diff
@@ -1159,7 +1159,7 @@ def get_aggregate_predictions(
         # total turnout predictions
         aggregate_z_total = (
             aggregate_z_unexpected + aggregate_z_train + aggregate_indicator_test.T @ self.weighted_z_test_pred
-        )
+        ).flatten()
```

Comment:
no issue, but curious why this change?

Reply:
`flatten()` on `aggregate_z_total` was being called twice, and I needed it to be called a third time, so I figured, let's just do it once 😄 🤷🏻‍♀️